sensor/kubernetes/eventpipeline/pipeline_test.go

```diff
@@ -183,7 +184,7 @@ func (s *eventPipelineSuite) Test_ReprocessDeployments() {
 		assert.True(s.T(), resourceEvent.DeploymentReferences[0].ForceDetection)
 	})
 
-	err := s.pipeline.ProcessMessage(msgFromCentral)
+	err := s.pipeline.ProcessMessage(context.TODO(), msgFromCentral)
```

Comment:
```suggestion
	err := s.pipeline.ProcessMessage(s.T().Context(), msgFromCentral)
```
Applies to the other calls in this file.

Reply:
[cfd53cb](https://github.com/stackrox/stackrox/pull/16064/commits/cfd53cb0dcd9792ef1e81c9bd64fb961897f897f)

Reply:
Sorry, I missed few places. Here is the part 2 of using the test context: [d3d5742](https://github.com/stackrox/stackrox/pull/16064/commits/d3d5742b7d4d36f771a355ab61fdea7911a9aa71)